features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -175,6 +183,34 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
         viewModel.allMerchantLocations.observe(viewLifecycleOwner) { merchantLocations ->
             merchantLocationsAdapter.submitList(merchantLocations)
         }
+
+        syncProgress.observe(viewLifecycleOwner) { syncProgress ->
+            if (syncProgress != null) {
```

Comment:
This doesn't look like it can be `null`

Reply:
This is a habit from DashPay and this check was removed.